src/Core/Components/Checkbox/FluentCheckbox.razor.cs

```diff
@@ -10,6 +11,16 @@ public partial class FluentCheckbox : FluentInputBase<bool>
     private bool? _checkState = false;
     private const string JAVASCRIPT_FILE = "./_content/Microsoft.FluentUI.AspNetCore.Components/Components/Checkbox/FluentCheckbox.razor.js";
 
+    protected override string? ClassValue
```

Comment:
Could you move this property to the end of the properties (after CheckStateChanged)?
We'll then have Constructors / Inject / Parameters / Public properties / Public methods / Private

See: https://dvoituron.com/naming-conventions/blazor/

Reply:
✅